persist/fs/index_lookup_test.go

```diff
@@ -99,3 +236,19 @@ func calculateExpectedDigest(t *testing.T, digestReader digest.FdWithDigestReade
 	require.NoError(t, err)
 	return digestReader.Digest().Sum32()
 }
+
+type sortableSummaries []schema.IndexSummary
+
+func (s sortableSummaries) Len() int {
+	return len(s)
+}
+
+func (s sortableSummaries) Less(i, j int) bool {
+	return bytes.Compare(s[i].ID, s[j].ID) < 0
+}
+
+func (s sortableSummaries) Swap(i, j int) {
+	temp := s[i]
+	s[i] = s[j]
+	s[j] = temp
```

Comment:
This can just be `s[i], s[j] = s[j], s[i]` btw.

Reply:
oh nice thanks